ui/pages/confirmations/hooks/send/useMaxAmount.ts

```diff
@@ -35,7 +40,7 @@ export const getEstimatedTotalGas = (gasFeeEstimates?: GasFeeEstimatesType) => {
     10,
   );
   const conversionrate = new Numeric(GWEI_TO_WEI_CONVERSION_RATE, 10);
-  return totalGas.times(conversionrate);
+  return totalGas.times(conversionrate).add(new Numeric(layer1GasFees, 16));
```

Comment:
Why do we need to convert to wei here before adding `layer1GasFees`? The maxFeePerGas * gasLimit should yield a result in wei directly.